### DISCUSSION TITLE
No module named 'MySQLdb'

### DISCUSSION THREAD

**Discussion by flowerboyworld** [sqlalchemy/sqlalchemy#8535] [Usage Questions]
Hi all,

I want to use SQLalchemy to use pandas.dataframe.to_sql() and pip installling sqlalchemy works but create_engine() throws the error ModuleNotFoundError: No module named 'MySQLdb'

Now from what I read MySQLdb is not running on python 3 and the few solutions I've found do not work for me. I'm working on a Mac and trying to connect to a freesqldatabase.com-databese.

**Comment by CaselIT** [comment]
Hi,

You will need to install one of the supported database driver for MySQL. See https://docs.sqlalchemy.org/en/14/dialects/mysql.html for details
